src/main/java/com/final_assignment/UpdateService.java

```diff
@@ -11,12 +12,12 @@ public UpdateService(UpdateMapper updateMapper) {
         this.updateMapper = updateMapper;
     }
 
-    public Entity update(Integer id, UpdateRequest request) {
-        Entity entity = updateMapper.update(id, request);
-        if (entity == null) {
-            // ユーザーが見つからない場合の処理
-            return null;
+    public boolean findByUpdateId(Integer id, String name, String email) {
+         Entity entity= updateMapper.findById(id);
```

Comment:
Entityという不明瞭な型でなくUserのようにデータの内容がクラス名からわかるようにしてほしい。
```suggestion
         User entity= updateMapper.findById(id);
```
EntityはあくまでもDBのレコードに対応するJavaのクラスの総称。